runtime/etc/init/planb-tokeninfo.conf

```diff
@@ -18,6 +18,11 @@ script
             export ENABLE_INTROSPECTION="true"
         fi
         if [ -n "$config_local_planb_tokeninfo_lightstep_token" ]; then
+            if [[ $config_local_planb_tokeninfo_lightstep_token == "aws:kms:"* ]]
+            then
+              config_local_planb_tokeninfo_lightstep_token=${config_local_planb_tokeninfo_lightstep_token##aws:kms:}
+              config_local_planb_tokeninfo_lightstep_token=`python3 /opt/taupage/bin/decrypt-kms.py $config_local_planb_tokeninfo_lightstep_token`
```

Comment:
Do you mind running it via `shellcheck` and fixing the issues? You can obviously ignore the warnings about undefined variables, but the rest is fixable.

Reply:
I ran the shellcheck and did some changes